lightning/src/ln/channelmanager.rs

```diff
@@ -10389,14 +10419,22 @@ where
 				}
 			},
 			#[cfg(async_payments)]
-			OffersMessage::StaticInvoice(_invoice) => {
-				match responder {
-					Some(responder) => {
-						responder.respond(OffersMessage::InvoiceError(
-								InvoiceError::from_string("Static invoices not yet supported".to_string())
-						))
-					},
+			OffersMessage::StaticInvoice(invoice) => {
+				let responder = match responder {
+					Some(responder) => responder,
 					None => return ResponseInstruction::NoResponse,
+				};
```

Comment:
The offers spec wasn't very clear on whether an invoice needs a reply path, so we don't rely on having one. Should we do the same for static invoices?

Reply:
Yeah, I ended up never replying if initiating an async payment to a static invoice fails. The only situation where we'd want to reply is if the invoice + invreq are valid and expected but missing an amount, which seems like it'll be rare and not worth doing some special error handling for.